index.html

```diff
@@ -179,34 +197,33 @@ <h2>Multi-Post Stories</h2>
 </div>
 </div>
 </section>
-<section id="about_myself">
-  <div class="myself">
-    <div>
+
+<!-- About -->
+<section id="about">
+  <div class="abt-1">  
+  <div class="about_myself">
+   
       <h1>About Myself</h1>
-    </div>
+   
     <div>
-      <p>
-        I'm a software developer! I can help you build a product, 
-        feature or website Look through some of my work and
-        experience! If you like what you see and have a project
-        you need coded, don't hesitate to contact me.
-      </p>
+      <p>I'm a software developer! I can help you build a product, 
+        feature or website Look through some of my work and experience! If 
+        you like what you see and have a project you need coded, don't 
+        hesitate to contact me.</p>
     </div>
     <div><p class="lets_connect">LET'S CONNECT</p></div>
-    <div>
       <ul class="social_icons">
-        <li><a href="https://github.com/katarighe/"><img class="icons" src="./images/Github.svg" alt="github" /></a></li>
-        <li><a href="https://linkedin.com/in/mighe/"><img class="icons" src="./images/Linkedin.svg" alt="Linkedin" /></a></li>
-        <li><a href="https://www.reddit.com/"><img class="icons" src="./images/hand.svg" alt="Hand" /></a></li>
-        <li><a href="https://twitter.com/AthlonWI/"><img class="icons" src="./images/Twitter.svg" alt="Twitter" /></a></li>
-        <li><a href="http://www.microverse.com/"><img class="icons" src="./images/M.svg" alt="Microverse" /></a></li>
+        <li><a href="https://github.com/katarighe/"><img class="icons" src="./images/icons/Github.svg" alt="github" /></a></li>
+        <li><a href="https://linkedin.com/in/mighe/"><img class="icons" src="./images/icons/Linkedin.svg" alt="Linkedin" /></a></li>
+        <li><a href="https://www.reddit.com/"><img class="icons" src="./images/icons/hand.svg" alt="Hand" /></a></li>
+        <li><a href="https://twitter.com/AthlonWI/"><img class="icons" src="./images/icons/Twitter.svg" alt="Twitter" /></a></li>
+        <li><a href="http://www.microverse.com/"><img class="icons" src="./images/icons/M.svg" alt="Microverse" /></a></li>
       </ul>
+      <div class="card_action">
+        <button class="secondary">Get my resume</button>
+      </div>
     </div>
-    <div class="card_action">
-      <button class="secondary">Get my resume</button>
-    </div>
-  </div>
-  <ul class="myskills">
+  <div class="myskills">
```

Comment:
- Please fix the About section of your desktop version it is not aligned as it is displayed in the Figma template. I have marked the sections to be corrected in the image below. Please fix this. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- I suggest aligning the technology section using the `flex` display property, you will find it easier to work with 😄 
- The dropdown arrows are not all the same as you have designed in your project, please study the figma template and make the required changes 👍 👍 
- There is a missing border-radius on the top right of this section, please fix this 👍 
![6is4u5](https://github.com/katarighe/my-portfolio/assets/73607512/fd8e23fb-3cb5-4945-86be-d31b72c1bced)

Design template:
![srtjsetjsetj](https://github.com/katarighe/my-portfolio/assets/73607512/de350ac3-f303-4f06-91ab-85cfcfa3e763)

Mobile version - Due to the changes made, there are some images displaying in the mobile version of this section, please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(mobile):
![wrgawrgaw](https://github.com/katarighe/my-portfolio/assets/73607512/5fd446b1-ad8c-41ab-834f-0385ebb8c701)